DelvUI/Interface/GeneralElements/HideHudConfig.cs

```diff
@@ -21,19 +21,23 @@ public class HideHudConfig : PluginConfigObject
         public bool HideOnlyJobPackHudOutsideOfCombat = false;
 
         [Checkbox("Hide Default Job Gauges", isMonitored = true)]
-        [Order(15)]
+        [CollapseControl(15, 0)]
         public bool HideDefaultJobGauges = false;
 
+        [Checkbox("I Dont Care About The Sound", isMonitored = true)]
+        [CollapseWith(0, 0)]
+        public bool IDontCareAboutTheSounds = false;
```

Comment:
The naming here is werid.
If disabling this boolean makes the sounds from default job bars not work at all, then change the name to something more intuitive like "Disable Job Gauges Sounds".
If disabling this boolean doesn't ensure anything, then I think it should ensure something haha.

Reply:
yeah it removes the sound by removing the NodeList from the Job Hud.

Reply:
OK then let's use a better name like "Disable Job Gauge Sounds"